scripts/gceworker.sh

```diff
@@ -32,13 +32,16 @@ case "${cmd}" in
            --network "default" \
            --maintenance-policy "MIGRATE" \
            --image-project "ubuntu-os-cloud" \
-           --image-family "ubuntu-1804-lts" \
+           --image-family "ubuntu-2004-lts" \
            --boot-disk-size "100" \
            --boot-disk-type "pd-ssd" \
            --boot-disk-device-name "${NAME}" \
            --scopes "cloud-platform"
     gcloud compute firewall-rules create "${NAME}-mosh" --allow udp:60000-61000
 
+    sudo rm ~/.ssh/config
```

Comment:
hmm, this smells like something i'm not sure i want a script to do for me.

Reply:
Stop reviewing hacky work around code dear friend

Reply:
Hehehe I was gonna try it!!!